android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -157,6 +166,24 @@ public void networkUnavailable() {
 
     }
 
+    private void showProgressBar(boolean show) {
+        if(show)
+            downloadProgressDialog.show();
+        else
+            downloadProgressDialog.dismiss();
+    }
+
+    private void setupProgressBar() {
+        downloadProgressDialog = new ProgressDialog(getContext());
+        downloadProgressDialog.setIndeterminate(true);
+        downloadProgressDialog.setProgressPercentFormat(null);
+        downloadProgressDialog.setProgressNumberFormat(null);
+        downloadProgressDialog.setProgressStyle(ProgressDialog.STYLE_HORIZONTAL);
+        downloadProgressDialog.setCancelable(false);
+        String shownMessage = String.format(getString(R.string.downloading_format), getString(R.string.menu_feed));
+        downloadProgressDialog.setMessage(shownMessage);
```

Comment:
That has been discussed in the threads above. As @iamareebjamal pointed out, the downloadCompleteHandler class was meant to be handling event download events and therefore it seemed it was not a good practice to use just the progress bar from that class when the work performed by both is poles apart.